### ISSUE TITLE
[Example Request]: Add seq2seq in eager examples

### ISSUE THREAD

**Issue by nthakor** [tensorflow/tensorflow#19115]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: NA
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: NA
- **TensorFlow installed from (source or binary)**: NA
- **TensorFlow version (use command below)**: NA
- **Python version**:  NA
- **Bazel version (if compiling from source)**: NA
- **GCC/Compiler version (if compiling from source)**: NA
- **CUDA/cuDNN version**: NA
- **GPU model and memory**: NA
- **Exact command to reproduce**: NA

existing tf.contrib.seq2seq in not compatible with eager mode. Can you add a seq2seq example with the eager mode? Something similar to [this](https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html).

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 17 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by bignamehyp** [comment]
It's a contrib project. 
Please contact the original author of seq2seq @alrojo
